apps/explorer/src/ui/stories/ProgrammableTransactionCommands.stories.tsx

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import {
+    type MoveCallSuiCommand,
+    type SuiArgument,
+    type SuiMovePackage,
+} from '@mysten/sui.js';
+import { type Meta, type StoryObj } from '@storybook/react';
+import { BrowserRouter } from 'react-router-dom';
+
+import { RpcClientContext } from '../../../../core';
+
+import {
+    Command,
+    CommandType,
+    type ICommandProps,
+} from '~/pages/transaction-result/programmable-transaction-view/Command';
```

Comment:
This story should be co-located here, not in the UI folder.

Reply:
removed the story for page